docs/source/ref/settings.rst

```diff
@@ -480,6 +480,19 @@ Default: ``False``
 Allows unicode characters in slugs generated by ``AutoSlugField``,
 which is supported by the underlying ``SlugField`` in Django>=1.9.
 
+
+Dynamic importer settings
+=========================
+
+``OSCAR_DYNAMIC_CLASS_LOADER``
+----------------------------------
+
+Default: not set
+
+A dotted Python path to custom callable to dynamically import classes, to replace
```

Comment:
I'd suggest the following alternative wording:

    Default: ``oscar.core.loading.default_class_loader``

    A dotted path to the callable used to dynamically import classes.

And then specify the value in `defaults.py`